application/controllers/HostsController.php

```diff
@@ -50,15 +50,19 @@ public function indexAction()
 
         $limitControl = $this->createLimitControl();
         $paginationControl = $this->createPaginationControl($hosts);
+
+        $defaultSort = 'host.state.severity desc,host.state.last_state_change desc';
         $sortControl = $this->createSortControl(
             $hosts,
             [
                 'host.display_name'                                          => t('Name'),
                 'host.state.severity desc,host.state.last_state_change desc' => t('Severity'),
```

Comment:
```suggestion
                'host.state.severity desc,host.state.last_state_change desc' => t('Severity'),
                'host.display_name'                                          => t('Name'),
```

You need to also change the order of this array to reflect that the hosts were ordered by severity by default.